features/admin.policy-administration.v1/components/policy-editor/policy-editor.tsx

```diff
@@ -0,0 +1,309 @@
+                                        </MenuItem>
+                                        <MenuItem value={ SupportedEditorThemes.DARK }>
+                                            { t("authenticationFlow:scriptEditor.themes.dark.label") }
+                                        </MenuItem>
+                                        <MenuItem value={ SupportedEditorThemes.HIGH_CONTRAST }>
+                                            { t("authenticationFlow:scriptEditor.themes.highContrast.label") }
+                                        </MenuItem>
+                                    </Select>
+                                </FormControl>
+                            </div>
+                        </>
+                    ) }
+                    <div className="editor-fullscreen">
+                        <Tooltip
+                            title={
+                                scriptEditorPanelSizeMode === ScriptEditorPanelSizeModes.Minimized
+                                    ? t("common:goFullScreen")
+                                    : t("common:exitFullScreen")
+                            }
+                            data-componentid="editor-fullscreen-toggle-tooltip"
+                        >
+                            <IconButton
+                                size="small"
+                                onClick={ handleScriptEditorPanelSizeChange }
+                            >
+                                {
+                                    scriptEditorPanelSizeMode === ScriptEditorPanelSizeModes.Minimized
+                                        ? <MaximizeIcon height={ 16 } width={ 16 } />
+                                        : <MinimizeIcon height={ 16 } width={ 16 } />
+                                }
+                            </IconButton>
+                        </Tooltip>
+                    </div>
+                </div>
+            </Toolbar>
+        </Box>
+    );
+
+    return (
+        <Suspense fallback={ <CircularProgress /> }>
+            <div className={ classNames("script-editor-panel", className) } data-componentid={ componentId }>
+                { ScriptEditorToolbar }
+                <Modal
+                    aria-labelledby="transition-modal-title"
+                    aria-describedby="transition-modal-description"
+                    open={ scriptEditorPanelSizeMode === ScriptEditorPanelSizeModes.Maximized }
+                    onClose={ handleScriptEditorPanelSizeChange }
+                >
+                    <Box className="full-screen-script-editor-container">
+                        { ScriptEditorToolbar }
+                        { ScriptEditor }
+                    </Box>
+                </Modal>
+                { ScriptEditor }
+            </div>
+        </Suspense>
+    );
+};
+
+/**
+ * Default props for the script editor panel component.
+ */
+PolicyEditorPanel.defaultProps = {
```

Comment:
Let's avoid using defaultProps as it is about to be deprecated. Let's use the following pattern to init default props instead.

https://github.com/wso2/identity-apps/blob/66cb2e19b5ba814e6e3dd9cad8fd094728ed237f/features/admin.application-templates.v1/components/application-creation-adapter.tsx#L59-L64